src/components/Avatar.tsx

```diff
@@ -0,0 +1,44 @@
+import { forwardRef } from 'react';
+import { useTheme } from '@mui/material/styles';
+import { Avatar as MUIAvatar, AvatarProps } from '@mui/material';
+
+// ----------------------------------------------------------------------
+
+type AvatarColor = 'default' | 'primary' | 'secondary' | 'info' | 'success' | 'warning' | 'error';
+
+// ----------------------------------------------------------------------
+
+export interface Props extends AvatarProps {
+  color?: AvatarColor;
+}
+
+const Avatar = forwardRef<HTMLDivElement, Props>(
+  ({ color = 'default', children, sx, ...other }, ref) => {
+    const theme = useTheme();
+
+    if (color === 'default') {
+      return (
+        <MUIAvatar ref={ref} sx={sx} {...other}>
```

Comment:
Are they using different svgs for the avatar to change colors?

Reply:
I have yet to discover all the details about the reusable components, not sure.